src/client/Client.cc

```diff
@@ -3711,8 +3731,9 @@ int Client::mark_caps_flushing(Inode *in)
   int flushing = in->dirty_caps;
   assert(flushing);
 
-  if (flushing && !in->flushing_caps) {
+  if (!in->flushing_caps) {
```

Comment:
On Fri, May 29, 2015 at 6:21 AM, Gregory Farnum <notifications@github.com>
wrote:

> In src/client/Client.cc
> <https://github.com/ceph/ceph/pull/4778#discussion_r31283951>:
>
> > @@ -3711,8 +3731,9 @@ int Client::mark_caps_flushing(Inode *in)
> >    int flushing = in->dirty_caps;
> >    assert(flushing);
> >
> > -  if (flushing && !in->flushing_caps) {
> > +  if (!in->flushing_caps) {
>
> You're just clearing out the flushing check here because we already
> asserted it above, or was there some other reason?
>
the assertion above is the reason I remove it


> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/4778/files#r31283951>.
>